[2024-04-12T12:10:33.745+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-12T12:10:33.859+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: prioritas2.write_to_csv manual__2024-04-12T12:08:19.276742+00:00 [queued]>
[2024-04-12T12:10:33.877+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: prioritas2.write_to_csv manual__2024-04-12T12:08:19.276742+00:00 [queued]>
[2024-04-12T12:10:33.878+0000] {taskinstance.py:2303} INFO - Starting attempt 2 of 6
[2024-04-12T12:10:33.924+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): write_to_csv> on 2024-04-12 12:08:19.276742+00:00
[2024-04-12T12:10:33.939+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'prioritas2', 'write_to_csv', 'manual__2024-04-12T12:08:19.276742+00:00', '--job-id', '167', '--raw', '--subdir', 'DAGS_FOLDER/prioritas2_dag.py', '--cfg-path', '/tmp/tmplgc2fgdx']
[2024-04-12T12:10:33.941+0000] {standard_task_runner.py:91} INFO - Job 167: Subtask write_to_csv
[2024-04-12T12:10:33.943+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=748) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-12T12:10:33.944+0000] {standard_task_runner.py:63} INFO - Started process 751 to run task
[2024-04-12T12:10:34.027+0000] {task_command.py:426} INFO - Running <TaskInstance: prioritas2.write_to_csv manual__2024-04-12T12:08:19.276742+00:00 [running]> on host 805e8d5936d1
[2024-04-12T12:10:34.147+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='abdan' AIRFLOW_CTX_DAG_ID='prioritas2' AIRFLOW_CTX_TASK_ID='write_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2024-04-12T12:08:19.276742+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-12T12:08:19.276742+00:00'
[2024-04-12T12:10:34.149+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-12T12:10:34.150+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-12T12:10:34.151+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: write_to_csv() missing 1 required positional argument: 'data'
[2024-04-12T12:10:34.167+0000] {taskinstance.py:1205} INFO - Marking task as UP_FOR_RETRY. dag_id=prioritas2, task_id=write_to_csv, execution_date=20240412T120819, start_date=20240412T121033, end_date=20240412T121034
[2024-04-12T12:10:34.188+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 167 for task write_to_csv (write_to_csv() missing 1 required positional argument: 'data'; 751)
[2024-04-12T12:10:34.199+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-04-12T12:10:34.225+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-12T12:10:34.231+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
